### DISCUSSION TITLE
Decoding Data

### DISCUSSION THREAD

**Discussion by hoffadg** [DSheirer/sdrtrunk#1470] [General]
Hi All,

I've been monitoring the MN Armer P25 system for a while now and I've noticed often times when Police are dispatched the dispatcher often implies that she/he is sending some information related to the call.  I've also heard them say "check your messages".  I'm wondering if data is also being sent along with the voice transmission and if it can be decoded.  If it isn't part of the P25 system, is there a separate "data only" system that is typically used?  I'd be interested in reading the messages if possible.  It would be really cool if SDRTrunk could do this in a separate window.

Thanks,
Dan

**Comment by Grey9498** [comment]
Usually when they say they are sending info over to a unit, they mean via a Mobile Data Terminal or MDT, the in-car laptop you see in patrol vehicles.

MDTs typically run on a closed loop cellular network.

To be honest, I don't know if it is possible for an SDR to scan cellular frequencies. Most (not all) modern cellular bands are outside of an SDR's normal range (1900MHz and above). The ones that are in an SDR's range are newer LTE and 5G bands, which are end to end encrypted.

You probably won't get in trouble for DECODING digital radio (P25, DMR, NXDN etc), but you probably WILL be in hot water if you are caught DECRYPTING encrypted data.

There is also a question of legality. In the US it is illegal to build a scanner that can intercept cellular frequencies... But technically and SDR isn't a scanner.

**Comment by Ulti-P-Uszer** [comment]
In the yr 2000, Bill Cheek got brought up on Federal changes for selling a kit of ICs & parts that he called a "data slicer" and it could decode the data that you are talking about. We don't know if Bill could have beaten the rap, by saying it was freely receivable like any other radio signal. Because Bill got cancer from having been in Vietnam 25 yrs earlier and died before his Federal trial could start. 

For those who might not know: Bill Cheek was a famous radio electronics engineer and author of many books on the subject, including 3 books on modifying police scanners. He truly was my kinda guy.

**Comment by dave-fl** [comment]
@Grey9498 Does the same apply for Fire with respect to their MDTs?  Trying to understand if those messages would be sent on a closed loop cell network or though a message on the P25 system that can be decoded.

**Reply by Grey9498** [reply]
I can't speak for everyone. However our department MDT's are on a closed loop Verizon network. Messages could be sent between terminals much like sending a PM on Facebook.

P25 does have the capability to send text messages unit to unit. But I couldn't say how sophisticated it's data transfer capabilities are (like sending photos or full reports and such).
